### PR TITLE
append tags and wait for api.url

### PR DESCRIPTION
**Pull request by yasun1**
close [OCPQE-15266](https://issues.redhat.com//browse/OCPQE-15266)
[OCPQE-17764](https://issues.redhat.com//browse/OCPQE-17764)

### CODE REVIEW

**Comment by yasun1** [comment]
/pj-rehearse pull-ci-openshift-aws-load-balancer-operator-main-e2e-aws-rosa-operator

**Comment by yasun1** [comment]
/pj-rehearse periodic-ci-openshift-jenkins-release-4.12-jenkins-hypershift4.12-aws-jenkins-lp-rosa-hypershift-412

**Comment by yasun1** [comment]
/pj-rehearse periodic-ci-openshift-jenkins-release-4.12-jenkins-hypershift4.12-aws-jenkins-lp-rosa-hypershift-412

**Comment by yasun1** [comment]
/pj-rehearse pull-ci-openshift-aws-load-balancer-operator-release-1.0-e2e-aws-rosa-operator

File: ci-operator/step-registry/rosa/cluster/provision/rosa-cluster-provision-commands.sh [hunk 1]
```diff
@@ -107,6 +107,11 @@ fi
 echo "Choosing openshift version ${OPENSHIFT_VERSION}"
 
 # Switches
+TAGS="prowci:${CLUSTER_NAME}"
```
Comment (yuwang-RH) [inline ci-operator/step-registry/rosa/cluster/provision/rosa-cluster-provision-commands.sh:110]: Tags should use the format Tags are comma separated, for example: 'key value, foo bar' as the help message shows. It is same to CLUSTER_TAGS. New update.

Comment (yuwang-RH) [reply ci-operator/step-registry/rosa/cluster/provision/rosa-cluster-provision-commands.sh]: @yasun1 One more thing, if the tags should be included with double quotation marks, or like --tags "a v,fd p:f,l:o pp"

**Pull request merged**